HL7-DaVinci/test-ehr
# Fix DeviceRequest Remove CPT code

This PR fixes the deviceRequest which doesn't contain insurance extension; also removes the CPT code which we don't support. 

To test the PR:

1. Using rest fo the repos in master branch
2. Select RAD (E40470) for William Oster, then the coverage should be prefetched, then submit will get back the cards as usual
